src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useContext } from 'react';
+import { DispatchContext, TodosContext } from '../../Store';
+
+export const Footer: React.FC = () => {
+  const {
+    todos,
+    selectedAll,
+    selectedActive,
+    selectedCompleted,
+  } = useContext(TodosContext);
+  const dispatch = useContext(DispatchContext);
+
+  const todosNotCompleted = [...todos].filter(todo => todo.completed === false);
```

Comment:
Unnecessary spread operator. You can filter the todos array directly, like so: todos.filter(todo => !todo.completed)